book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+
+class Book < Item
+  attr_reader :cover_state, :publisher
+
+  def initialize(publisher, cover_state, publish_date)
+    super(nil, nil, nil, nil, publish_date)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
```

Comment:
Update the super file to match the new Item.rb file the and that way we don't override other instance variables.